src/main/webapp/city.js

```diff
@@ -3,9 +3,11 @@
  */
 async function searching(id) {
   const cityResponse = await fetch("/api/cities/" + id);
+  const countryResponse = await fetch("/api/countries/" + city.countryId);
   const city = await cityResponse.json();
```

Comment:
Gotta switch these two lines
```suggestion
  const city = await cityResponse.json();
  const countryResponse = await fetch("/api/countries/" + city.countryId);
```